client/components/section/index.tsx

```diff
@@ -19,7 +19,7 @@ interface SectionHeaderProps {
 }
 
 export const SectionContainer = styled.div< SectionContainerProps >`
-	::before {
+	.theme-default:not( .is-global ) &::before {
```

Comment:
Instead of adding the global restriction here, I could add `position: relative` to the main element. It seems to work the same in terms of fixing this, but Im not sure if it may cause regressions elsewhere. Im not sure which fix (or if something else) is better.